src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -346,10 +346,10 @@ internal override void RequestRun()
                 // We will be needing a separate variable(boolean flag) to check whether run can be enabled from external applications
                 // and not confuse it with the internal flag RunEnabled which is associated with the Run button in Dynamo. 
                 // Make this RunSettings.RunEnabled private, introduce the new flag and remove the "executingTask" variable. 
-                if (RunSettings.RunEnabled || executingTask)
+                if ((RunSettings.RunEnabled || executingTask) && !DelayedGraphExecution.IsEnabled)
```

Comment:
This being static makes this simple, but it's another static property we would need to remove if we want to move towards multiple home workspaces or multiple Dynamo models...

Reply:
Oh, I did not consider that.
I think we can make it a protected member of `WorkspaceModel`